pygfx/helpers/_gizmo.py

```diff
@@ -3,12 +3,12 @@
 """
 
 import numpy as np
+import pylinalg as la
```

Comment:
Lots of changes were needed here. I'll give the gizmo example a good spin to see if it still works as intended. (Using this comment as a reminder.)